py/aou_workbench_client/data.py

```diff
@@ -50,24 +53,23 @@
     filters and concept_set_name / concept_ids / source_concept_ids are specified,
     rows returned must match both.
   :param cohort_statuses: a list of CohortStatus indicating a filter on the review status of 
-    participants to be returned in the resulting data table; defaults to no filtering (all 
-    participants are returned.
+    participants to be returned in the resulting data table; defaults to all non-excluded participants
+    being returned.
   :param max_results: the maximum number of rows to return in the resulting data table; defaults
     to no limit (all matching rows will be returned.) Note that for large cohorts, it may take a 
     long time to get all results.
-  :param order_by: a list of column names from the table or related tables to order the results by; 
+  :param order_by: a list of column names from the table or related tables to order the results by,
+    any of which can be optionally wrapped in `DESCENDING()` to request descending sort order; 
     defaults to order by person_id and primary key of the specified table; e.g. 
     [Person.gender_concept_id, Person.person_id]
-  :param page_size: the maximum number of result rows to fetch in a single API request when 
-    retrieving results; defaults to 1000.
   :param debug: true if debug request and response information should be displayed; defaults to 
     false.  
-  :return a generator of dictionaries representing the results to the query
+  :return a Pandas data frame representing the results of the query
 """
-def load_data_table(cohort_name, table, columns=None, concept_set_name=None, concept_ids=None,
+def load_data_frame(cohort_name, table, columns=None, concept_set_name=None, concept_ids=None,
```

Comment:
nit: naming seems slightly asymmetric with `load_data_frame()` and `load_annotations` (both returning data frames), should this just be called `load` or `load_data`?